docs/source/tutorials/building_blocks.rst

```diff
@@ -22,15 +22,15 @@ Mobjects
 
 Mobjects are the basic building block for all manim animations.  Each class
 that derives from :class:`.Mobject` represents an object that can be displayed
-on screen.  For example, simple shapes such as :class:`.Circle`,
+on the screen.  For example, simple shapes such as :class:`.Circle`,
 :class:`.Arrow`, and :class:`.Rectangle` are all mobjects.  More complicated
 constructs such as :class:`.Axes`, :class:`.FunctionGraph`, or
 :class:`.BarChart` are mobjects as well.
 
-If you try to display on screen an instance of :class:`.Mobject`, you will only
+If you try to display on the screen an instance of :class:`.Mobject`, you will only
 see an empty frame.  The reason is that the :class:`.Mobject` class is an
 abstract base class of all other mobjects, i.e. it does not have any
-pre-determined visual shape that can be displayed on screen.  It is only the
+pre-determined visual shape that can be displayed on the screen.  It is only the
```

Comment:
`on screen` seems perfectly fine in all these contexts.

Reply:
Check above comment